### ISSUE TITLE
label buttons for screen reader accessibility.

### ISSUE THREAD

**Issue by nktech1135** [Requarks/wiki#2154]
**Describe the bug**
All button elements in wiki.js need labels so a screen reader can tell what the button does.
As it is now, for screen reader users anything that shows to the screen reader as a button does not have labels.
examples are the page actions menu, the new page button, ETC.
Currently going to such an element just says button without giving the user any way of knowing what the button does.
Could this be fixed?

**Host Info (please complete the following information):**
 - OS: ubuntu 20.04
 - Wiki.js version: 2.4.107
 - Database engine: postgres 12.2

**Additional context**
the screen reader in question is nvda and operating system is windows 10.

**Comment by NGPixel** [comment]
Thanks for raising this issue! I assumed Vuetify (the UI framework) was handling icon buttons labelling automatically but that's apparently not the case. I added aria-labels to all buttons on the page view, as reported by the chrome lighthouse accessibility tool.

There're many more places around the app that needs to be addressed but this will be done gradually over time.

On a side note, I tried the NVDA app but I couldn't get the icon buttons to be read aloud. However, clicking on them triggers the read aloud just fine (and the aria-label is correctly added). I'm I missing something or using the screen reader wrong?

**Comment by nktech1135** [comment]
When using nvda you use the arrow keys to read the buttons, not the mouse.
If this was pushed to github i could try it out for you.

**Comment by NGPixel** [comment]
You can use commit e5ea4af089c11cfd5ec63e73d373c26474953bd1 or the docker image `requarks/wiki:canary-2.5.77` for testing.

**Comment by nktech1135** [comment]
I messed around with this for a while. Getting it to run took me a while as i didn't have a release version and i wasn't sure what wouldn't work if i cloned from github directly.
I finally got the setup to run and created a welcome page but got an error when i hit save. So far the buttons were still not labeled.

**Comment by nktech1135** [comment]
An update. I'm not sure if it's something on the screen reader side or if wiki.js is pulling updated assets, but some buttons do have labels now.

If you need more help just let me know.

Thanks.

**Comment by nktech1135** [comment]
OK, I finally updated to the latest stable version as i didn't have a clue how to build this from scratch. Things do work much better now. Most buttons have labels now.
Thanks for being responsive and fixing this. I really do enjoy using wiki.js and a good support experience makes this just that much better.
